### PR TITLE
If the citizenship is lost, mid-year change is allowed (and required)…

### LINKED ISSUE #240: Necessary mid-year citizenship change
2e2 should also allow someone to change their represented country in the middle of the year if they actually lost the former citizenship and thus _can't_ represent the former country for the rest of the year.

### PR DESCRIPTION
**Pull request by Claster**
If the citizenship is lost, mid-year change is allowed (and required). Fix #240.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
 - 2e) Competitors must represent a country of which they hold citizenship. The WCA Delegate should verify citizenship standing by means of documents (e.g. a passport) at their first competition. If a competitor is found ineligible to represent the country under which they have registered, the competitor may be disqualified retroactively and/or suspended, at the discretion of the WCA Board.
     - 2e1) The eligible countries are defined by the [WCA List of Recognized Countries](https://www.worldcubeassociation.org/regulations/countries/).
-    - 2e2) Competitors with updates to their citizenship status may change their country of representation in their first competition of a calendar year.
+    - 2e2) Competitors with updates to their citizenship status may change their country of representation in their first competition of a calendar year. Exception: if the citizenship of the country which the competitor represents is lost, they must change the country of representation at their next competition.
```
Comment (AlbertoPdRF) [inline wca-regulations.md]: We first say "in their first competition" and then "at their next competition", which preposition is correct in this context?

Comment (AlbertoPdRF) [inline wca-regulations.md]: We first say "in their first competition" and then "at their next competition", which preposition is correct in this context?

Comment (Claster) [reply wca-regulations.md]: Hmm. @lgarron, @kingmathyall, would you help, please?

Comment (jfly) [reply wca-regulations.md]: Definitely "at"!

**Comment by AlbertoPdRF** [comment]
I just made the change, thanks @jfly!

Also, most people won't change their nationality at their next competition, but e-mail their Delegate or the WRT to change it before their next competition. Should we add "or before" to make it clear that this can happen?

**Comment by Laura-O** [comment]
> Also, most people won't change their nationality at their next competition, but e-mail their Delegate or the WRT to change it before their next competition. Should we add "or before" to make it clear that this can happen?

Yes, that sounds good!

File: wca-regulations.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
 - 2e) Competitors must represent a country of which they hold citizenship. The WCA Delegate should verify citizenship standing by means of documents (e.g. a passport) at their first competition. If a competitor is found ineligible to represent the country under which they have registered, the competitor may be disqualified retroactively and/or suspended, at the discretion of the WCA Board.
     - 2e1) The eligible countries are defined by the [WCA List of Recognized Countries](https://www.worldcubeassociation.org/regulations/countries/).
-    - 2e2) Competitors with updates to their citizenship status may change their country of representation in their first competition of a calendar year.
+    - 2e2) Competitors with updates to their citizenship status may change their country of representation at their first competition of a calendar year. Exception: if the citizenship of the country which the competitor represents is lost, they must change the country of representation at their next competition.
```
Comment (lgarron) [inline wca-regulations.md]: Suggestion: "Exception: if the competitor loses citizenship of the country which they were previously representing, they must change their country of representation at their next competition."

**Comment by AlbertoPdRF** [comment]
For consistency, I would also add the "before or" to the first sentence.

**Pull request merged**